lib/rataextra-backend.ts

```diff
@@ -87,6 +88,42 @@ export class RataExtraBackendStack extends NestedStack {
       securityGroups: securityGroups,
     };
 
+    /**
+     * @param {string} projectRoot
+     * @returns {import('esbuild').Plugin}
+     */
+    const createFixDirnamesPlugin = (projectRoot: string) => {
```

Comment:
This and ESM_REQUIRE_SHIM are a bit much to add in the middle of this infra code. I'd suggest moving these elsewhere and importing them from there.